src/generic/graphicc.cpp

```diff
@@ -260,6 +260,13 @@ class wxCairoPenBrushBaseData : public wxGraphicsObjectRefData
 
     virtual void Apply( wxGraphicsContext* context );
 
+    void CreateLinearGradientBrush(wxDouble x1, wxDouble y1,
```

Comment:
AFAICS this function (and the one below) is used for pens too, so the name is now misleading. What about `CreateXXXGradientPattern()` instead?

Reply:
Sounds good. I was also thinking that a good idea to change those names